From 50507d6fc80117c3448029e4bd335704cd5e58ec Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Thu, 23 Jul 2026 20:25:54 +0300
Subject: [PATCH 06826/15333] apparmor: switch website link to https

Should make harder for MITM to redirect to somewhere else.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
Signed-off-by: John Johansen <john.johansen@canonical.com>
---
 security/apparmor/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/apparmor/Kconfig b/security/apparmor/Kconfig
index 1e3bd44643da..e1bfd164a23a 100644
--- a/security/apparmor/Kconfig
+++ b/security/apparmor/Kconfig
@@ -11,7 +11,7 @@ config SECURITY_APPARMOR
 	  This enables the AppArmor security module.
 	  Required userspace tools (if they are not included in your
 	  distribution) and further information may be found at
-	  http://apparmor.wiki.kernel.org
+	  https://apparmor.wiki.kernel.org
 
 	  If you are unsure how to answer this question, answer N.
 
-- 
2.55.0

